pyproject.toml

```diff
@@ -41,7 +41,6 @@ profile = "black"
 
 [tool.poetry.scripts]
 unblob = "unblob.cli:main"
-unromfs = "unblob.extractors.unromfs:cli"
```

Comment:
Just a question: Is it okay for us (=the user) that the `unromfs` command will no longer be available?

Reply:
Totally ok with this right now. If someone wants to extract a romfs file they can run unblob on it with depth set to 1 or 2. Check my other comments before merging.